config/common/server.yaml

```diff
@@ -0,0 +1,267 @@
+        deploymentConfig: "{{.ApplicationName}}-kieserver-{{$index}}"
+      template:
+        metadata:
+          name: "{{.ApplicationName}}-kieserverblh-{{$index}}"
+          labels:
+            deploymentConfig: "{{.ApplicationName}}-kieserver-{{$index}}"
+            app: "{{.ApplicationName}}"
+            service: "{{.ApplicationName}}-kieserver-{{$index}}"
+        spec:
+          serviceAccountName: "{{.ApplicationName}}-rhpamsvc"
+          terminationGracePeriodSeconds: 60
+          containers:
+            - name: "{{.ApplicationName}}-kieserver"
+              image: "rhpam{{.Version}}-kieserver-openshift"
+              imagePullPolicy: Always
+              resources:
+                requests:
+                  memory: 512Mi
+                limits:
+                  memory: 1Gi
+              volumeMounts:
+                - name: kieserver-keystore-volume
+                  mountPath: "/etc/kieserver-secret-volume"
+                  readOnly: true
+              livenessProbe:
+                exec:
+                  command:
+                    - "/bin/bash"
+                    - "-c"
+                    - curl --fail --silent -u adminUser:"$KIE_ADMIN_PWD" http://localhost:8080/services/rest/server/healthcheck
+                initialDelaySeconds: 180
+                timeoutSeconds: 2
+                periodSeconds: 15
+                failureThreshold: 3
+              readinessProbe:
+                exec:
+                  command:
+                    - "/bin/bash"
+                    - "-c"
+                    - curl --fail --silent -u adminUser:"$KIE_ADMIN_PWD" http://localhost:8080/services/rest/server/readycheck
+                initialDelaySeconds: 60
+                timeoutSeconds: 2
+                periodSeconds: 30
+                failureThreshold: 6
+              ports:
+                - name: jolokia
+                  containerPort: 8778
+                  protocol: TCP
+                - name: http
+                  containerPort: 8080
+                  protocol: TCP
+                - name: https
+                  containerPort: 8443
+                  protocol: TCP
+              env:
+              - name: KIE_ADMIN_USER
+                value: adminUser
+              - name: KIE_ADMIN_PWD
+                value: "{{.AdminPassword}}"
+              - name: KIE_SERVER_CONTROLLER_USER
+                value: controllerUser
+              - name: KIE_SERVER_CONTROLLER_PWD
+                value: "{{.ControllerPassword}}"
+              - name: KIE_SERVER_USER
+                value: executionUser
+              - name: KIE_SERVER_PWD
+                value: "{{.ServerPassword}}"
+              - name: DROOLS_SERVER_FILTER_CLASSES
+                value: "true"
+              - name: RHPAMCENTR_MAVEN_REPO_PATH
+                value: /maven2/
+              - name: KIE_SERVER_CONTROLLER_SERVICE
+                value: "{{.ApplicationName}}-rhpamcentr"
+              - name: KIE_MBEANS
```

Comment:
i believe there are a few env vars that aren't common here... including this MBEANS.